sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/JsonSupport.java

```diff
@@ -39,6 +39,7 @@
 import com.google.protobuf.UnsafeByteOperations;
 import kalix.javasdk.annotations.Migration;
 import kalix.javasdk.impl.ByteStringEncoding;
+import org.jetbrains.annotations.NotNull;
```

Comment:
removed, ale content type is once again optional.

Reply:
Using optional will be annoying for Scala and for users as well. 

Since the API is controlling the payload already, we can always set the content-type behind the scene. 

* when user passes `String` it will be `text/plain`
* when user passes `Object` it will be `application/json`
* when user passes `bytes[]` it will be `application/octet-stream`

The `NoContent` one can default to `application/octet-stream` which is not wrong since we are sending `byte[0]`.

Reply:
ok

Reply:
Jetbrains dependency still being imported here.

Reply:
removed